### ISSUE TITLE
'Jump to' links not working when selecting one-page option

### ISSUE THREAD

**Issue by lguariento** [Daniel-KM/Omeka-S-module-Reference#17]
Hello,

when deselecting the 'one page' option here:

![Screenshot 2022-09-23 at 14-30-51 Edit · Browse by original language · Pages · BOSLIT](https://user-images.githubusercontent.com/10143842/191971411-42b7b244-7507-449b-8ba0-83f08eb5cb6c.png)

The link to each letter in the list of letters on the top:

![Screenshot 2022-09-23 at 14-27-31 Browse by original language · boslit · BOSLIT](https://user-images.githubusercontent.com/10143842/191970838-f013facd-6407-4031-a22e-e2b4a7cb53ea.png)

doesn't do anything. I suspect this is because the hyperlinks point to a lowercase letter, whilst the `id`s of the anchors in the page are in capital letter.

```
<a href="#c">C</a>

<h3 class="reference-heading" id="C"><a href="#reference-skiplinks">C</a></h3>
```

Can these be lowercase letters if the 'one page' option is selected?

**Comment by Daniel-KM** [comment]
Thanks, this is fixed in last  version (3.4.38.3). Can you try it?

**Comment by lguariento** [comment]
Yes indeed, it's now fixed, thanks.
